packages/transforms/src/evade-censor.js

```diff
@@ -0,0 +1,55 @@
+// @ts-check
+
+/**
+ * Provides functions for evading SES restrictions.
+ *
+ * @module
+ */
+
+/**
+ * Matches `import(...)` expressions in comments (e.g., TS types in JSDoc)
+ */
+const IMPORT_RE = new RegExp('\\b(import)(\\s*(?:\\(|/[/*]))', 'sg');
+
+/**
+ * Matches the start of an HTML comment in a JS comment
+ */
+const HTML_COMMENT_START_RE = new RegExp(`${'<'}!--`, 'g');
+
+/**
+ * Matches the end of an HTML comment in a JS comment
+ */
+const HTML_COMMENT_END_RE = new RegExp(`--${'>'}`, 'g');
+
+/**
+ * Rewrites a Comment Node to avoid triggering SES restrictions.
+ *
+ * Apparently coerces all comments to block comments.
+ * @param {import('@babel/types').Comment} node
+ * @param {LocationUnmapper} [unmapLoc]
+ */
+export function transformComment(node, unmapLoc) {
+  node.type = 'CommentBlock';
+  // Within comments...
+  node.value = node.value
+    // ...strip extraneous comment whitespace
+    .replace(/^\s+/gm, ' ')
+    // ...replace HTML comments with a defanged version to pass SES restrictions.
+    .replace(HTML_COMMENT_START_RE, '<!X-')
+    .replace(HTML_COMMENT_END_RE, '-X>')
+    // ...replace import expressions with a defanged version to pass SES restrictions
+    // (featuring homoglyphs for @kriskowal)
+    .replace(IMPORT_RE, '\u{406}\u{16D6}\u{420}\u{39F}\u{13D2}\u{422}$2')
```

Comment:
summoning @mhofman, resident homoglyph idea haver. (I do not need credit)

Reply:
Is there a reason to use homoglyphs in addition to, uh, _lulz_?

Reply:
They have the benefit of preserving the column numbers for subsequent code on the same line. But mostly lulz.